lib/remont/attribute.rb

```diff
@@ -16,7 +16,7 @@ def initialize(name, opts = {}, &processor)
     # @param [ActiveRecord::Base] record
     # @return [Object]
     def process(value, record)
-      processor.call(value, record)
+      processor.call(name, value, record)
```

Comment:
I think `name` can be provided when the processor is assigned to the attribute. Can you provide an example of how you intend to use `name` in the attribute processor?

Since I'd consider the `name` to be the least important argument, it should be the last one in the argument list (so it can be omitted when declaring the block which doesn't use it)

If we decide on forwarding meta information, maybe it makes more sense to forward attribute reference instead of just the `name` value.

Reply:
I had a fleeting thought when it seemed like it would be necessary to have `name` available to be able to fetch the value of the attribute that contain the info on the file in the S3 bucket, but `value` is already available and it can actually be used for this from the start, and if `name` does turn out to be necessary, I'll see if I can provide it to the processor when it is assigned to the attribute.
I'll remove this change for now.